python/pyproject.toml

```diff
@@ -13,16 +13,52 @@
 # limitations under the License.
 
 [build-system]
-
+build-backend = "setuptools.build_meta"
 requires = [
     "wheel",
-    "setuptools",
+    "setuptools>=61.0.0",
     "cython>=0.29,<0.30",
     "scikit-build>=0.13.1",
     "cmake>=3.23.1,!=3.25.0",
     "ninja",
-    "cuda-python>=11.7.1,<12.0"
+    "cuda-python>=11.7.1,<12.0",
+    "tomli; python_version < '3.11'",
+]
+
+[project]
+name = "rmm"
+version = "23.04.00"
+description = "rmm - RAPIDS Memory Manager"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { text = "Apache 2.0" }
+requires-python = ">=3.8"
+dependencies = [
+    "cuda-python>=11.7.1,<12.0",
+    "numpy>=1.19",
+    "numba>=0.49",
 ]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Topic :: Database",
+    "Topic :: Scientific/Engineering",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
```

Comment:
Do we want to list Python 3.9? (Dropped from our package matrix in 23.02)

Reply:
Technically it should work. We only dropped it due to CI limitations. We do test on either end (Python 3.8 & 3.10). Would suggest leaving it

Reply:
Yeah, I noticed that as well. Sure, I can remove it in this PR.